### DISCUSSION TITLE
[A-L] (18 maggio 2024) Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/progettazione-di-algoritmi#95] [Esercitazioni A-L]
![immagine](https://github.com/user-attachments/assets/0528ecbc-4975-47f1-b3fa-c71be7b4c6ba)

**Comment by daromacarlo** [comment]
![immagine](https://github.com/user-attachments/assets/16cc605a-e8f7-4714-9dcd-47ba7da52948)
T(n) = O(M(n+m))
